bundler.d/journald.rb

```diff
@@ -0,0 +1,4 @@
+# disable to avoid journald native gem in development setup
+group :journald do
+  gem 'logging-journald', '~> 1.0'
```

Comment:
Looks like this causes an issue on Ruby 2.0:

```
    logging-journald (~> 1.0) was resolved to 1.0.0, which depends on
      ruby (>= 2.1.0)
```

Reply:
Thanks for heads up, I filed PR upstream it is a small change.

Reply:
So I removed Ruby 2.1 requirement in logging-journald and also pinned to journald-logger version 1.1.1 which still supports Ruby 2.0: https://github.com/lzap/logging-journald/commit/a58235e5c0f15370119442bd3cde3aa8379ddbc1

This will enable Jenkins/Travis to pass on Ruby 2.0. In RPM installations we don't use Bundler therefore the requirement for Ruby 2.1 in journald-logger 2.x does not matter as it's used in gemspec. It should work fine, the only reason for Ruby 2.1 is a method `real_exception` which is not used at all by my gem and if needed, we can patch RPM version to support it. For Debian we don't care as it uses Ruby 2.1 and 2.3 already.

Reply:
journald-logger seems to have fixed this now?